examples/lock-app/esp32/main/CMakeLists.txt

```diff
@@ -147,6 +147,7 @@ idf_component_register(PRIV_INCLUDE_DIRS
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/zzz_generated/lock-app/"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/examples/providers"                      
                       "${CMAKE_CURRENT_LIST_DIR}/include"
+		      "${IDF_PATH}/components/freertos/include/freertos"
```

Comment:
Why add this path to the PRIV_INCLUDE_DIRS? Can we add the freertos to the PRIV_REQUIRES components of the main component?

Reply:
Updated